GTBitsOfGood/southface
# rd/90-recent-standard-section

## Recent Standard Section

Issue Number(s): #90 

What does this PR change and why?
Added achieved project plans to the User and prepopulated archived project plan data

### Checklist

- [x] `archivedProjectPlans` property is added to the User Schema (see other notes).
- [x] pre populated archived project plan data is added. Do this from mongodb compass (check other notes). 
       - add pre-populated data to testAdmin and testNonAdmin user documents.
- [x] button is added in the `/plan-builder` page called "See all archived project plans"
- [x] onClick on button leads to `/saved-project-plans` page (tip: wrap the button in a next.js Link component)
- [x] on the saved project plans builder page, the current saved project plans (pre-populated data) is logged on the console


### How to Test

- Go to Project Plan Builder and Click on "See all archived project plans"

###Comments
- I was just wondering if "Archived Project Plan" and "Saved Project Plans" are the same thing. I assume they are so I removed some of the old code that was in saved-project plan. In case I was wrong, I left the button and only commented out the old code so that it can be added back if needed.
